_includes/navbar.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
-  <div class="navbar-header pull-left">
+  <div class="navbar-logo navbar-header pull-left">
     <a class="navbar-brand" href="{{ site.baseurl }}/">
-      <img src="{{ site.baseurl }}/img/Logo.png">
+      <img width="auto" height="26" src="{{ site.baseurl }}/img/Logo.png">
```

Comment:
I don't think `width="auto"` is valid HTML. The attribute only takes a value (in pixels), according to the relevant MDN documentation.

Reply:
@Skaty 

It seemed that I didnt need to specify the width, I have removed that and also the .vs files ;)